janet-http/http-service/src/main/java/io/techery/janet/HttpActionService.java

```diff
@@ -74,7 +75,8 @@ public HttpActionService(String baseUrl, HttpClient client, Converter converter)
         this.baseUrl = baseUrl;
         this.client = client;
         this.converter = converter;
-        this.runningActions = new CopyOnWriteArrayList<Object>();
+        this.actionHelperCache = new HashMap<Class, ActionHelper>();
+        this.runningRequests = new ConcurrentHashMap<Object, List<Request>>();
```

Comment:
Micro optimization for `ConcurrentHashMap` with initial params
https://ria101.wordpress.com/2011/12/12/concurrenthashmap-avoid-a-common-misuse/